arbitrum-docs/how-arbitrum-works/02-transaction-lifecycle.mdx

```diff
@@ -111,8 +136,8 @@ A dispute consists of two staked validators dissecting their disagreement down t
 
 **See:**
 
-- [Challenges](/how-arbitrum-works/fraud-proofs/challenge-manager.mdx)
-- [Wasm/WAVM](/how-arbitrum-works/fraud-proofs/wasm-wavm.mdx)
+- [Challenges](/how-arbitrum-works/07-interactive-fraud-proofs.mdx#challengemanager)
+- [Wasm/WAVM](/how-arbitrum-works/07-interactive-fraud-proofs.mdx#wasm-to-wavm)
 
 L1 contracts also keep track of the tree of all assertions; i.e., how many stakers are in disagreement, who is currently disputing with whom, etc. We refer to this level of Arbitrum's design architecture as its "assertion tree protocol."
```

Comment:
The "Assertion Tree Protocol" just below this paragraph is pointing at the page in the old structure, which doesn't have a sidebar.
We should move all these pages to a separate folder (for example `website/archive`), or remove them.

Reply:
will archive, update links